### ISSUE TITLE
Subdomain => 404

### ISSUE THREAD

**Issue by tbrunain** [grocy/grocy#106]
Hello,

I'm trying to set up grocy (docker-compose way) with Traefik to handle a subdomain (grocy.xxxx.xx) .
I can start grocy but I get a 404 on all the resources , the main html file is fine though and I see an ugly html , links are working and login as well.

I'm sure I missed a stupid config somewhere but can't figure out where :x

**Comment by berrnd** [comment]
Maybe @talmai can help, also watch #101 - the docker part will be moved to [grocy/grocy-docker](https://github.com/grocy/grocy-docker) anytime soon...

**Comment by talmai** [comment]
I'll be finally able to get docker moved to the new repo later this week (apologies for the delay @berrnd). 

@tbrunain I'm assuming you are stopping nginx and running traefik instead? Otherwise, wouldn't this be conflicting? I haven't used Traefik, so may only be able to provide minimal help there.

**Comment by berrnd** [comment]
> I'll be finally able to get docker moved to the new repo later this week (apologies for the delay @berrnd).

Absolutely no problem, it's great that you care about Docker for grocy! :)

**Comment by tbrunain** [comment]
It's strange, I've checked the error_logs of Nginx and here is what I found :
`2018/11/07 19:52:24 [error] 181#181: *3569 open() "/www/public/js/grocy.js" failed (2: No such file or directory), client: 172.18.0.2, server: grocy.xxxx.xxx, request: "GET /js/grocy.js?v=1.22.0 HTTP/1.1", host: "grocy.xxxx.xx"
`

But if I check what's inside grocy docker container, at that path (/www/public/js/grocy.js) , the file exist , and has - what I think - correct permissions .

(I'm using Traefik + Nginx , since Traefik is just a reverse proxy / load balancer and not a webserver) .

**Comment by blakkmaan** [comment]
You can use Nginx for reverse proxying as well. I would try that and I think this will resolve the issue. I think Nginx is looking for the file in his own www homedirectory and not in the docker volume .

**Comment by tbrunain** [comment]
Yeah sure , it's just that my setup at the moment is running behind traefik so I would like to keep using it : ) I haven't got time to continue to look for a solution ,if I found one I'll keep you informed !
